android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -386,6 +391,9 @@ private void refreshDetails()
     refreshWiFi();
     refreshMetadataOrHide(mMapObject.getMetadata(Metadata.MetadataType.FMD_FLATS), mEntrance, mTvEntrance);
     refreshMetadataOrHide(mMapObject.getMetadata(Metadata.MetadataType.FMD_LEVEL), mLevel, mTvLevel);
+    refreshMetadataOrHide(mMapObject.hasAtm() ?
+                          getString(R.string.type_amenity_atm) + ": " + getString(R.string.yes_available):
```

Comment:
Avoid manually concatenating strings, some languages may not use ':' or use a different symbol for it.